viral/offline_sequence_detection.py

```
@@ -0,0 +1,707 @@
+            if os.path.exists(
+                f"data/cache/{mouse_name}_{date}_online_en_bloc_decoding.npz"
+            ):
+                with np.load(
+                    f"data/cache/{mouse_name}_{date}_online_en_bloc_decoding.npz"
+                ) as npz:
+                    positions = npz["positions"]
+                    pr_max = npz["pr_max"]
+            else:
+                try:
+                    positions, pr_max = main(mouse_name, date)
+                except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)